src/js/providers/tracks-mixin.js

```diff
@@ -48,7 +48,7 @@ function setTextTracks(tracks) {
     } else {
         // Remove the 608 captions track that was mutated by the browser
         this._unknownCount = 0;
-        this._textTracks = _.reject(this._textTracks, function(track) {
+        this._textTracks = reject(this._textTracks, function(track) {
```

Comment:
We can use `.filter` here instead of `reject`. Reject is just `filter` which removes an item if `true` is returned, as opposed to false.